v23.1/show-backup.md

```diff
@@ -77,6 +77,7 @@ Field | Description
 `rows` | Number of rows in tables that are part of the backup.
 `create_statement` | The `CREATE` statement used to create [table(s)](create-table.html), [view(s)](create-view.html), or [sequence(s)](create-sequence.html) that are stored within the backup. This displays when `SHOW BACKUP SCHEMAS` is used. Note that tables with references to [foreign keys](foreign-key.html) will only display foreign key constraints if the table to which the constraint relates to is also included in the backup.
 `is_full_cluster` |  Whether the backup is of a full cluster or not.
+`regions` | Whether the database is multi-region or not. If the database is a multi-region database, a string of [`ALTER DATABASE`](alter-database.html) commands will show. If the database is single-region, `NULL` will show. 
```

Comment:
I'd rephrase this like the `create_statement` explanation above: "The `ALTER DATABASE` statements used to configure the database with multi region zone configurations, if they exist".

This link may be more appropriate: https://www.cockroachlabs.com/docs/dev/multiregion-overview.html#database-regions